lib/modules/datasource/aws-rds/readme.md

```diff
@@ -0,0 +1,76 @@
+This datasource returns the database engine versions available for use on [AWS RDS](https://aws.amazon.com/rds/) via the AWS API.
+These generally track publicly released database versions, though new versions may not be available on RDS for a few weeks or months after their release while AWS tests them, and existing versions may be pulled by AWS if serious problems arise during their use.
```

Comment:
This sentence is too long. Also I'm not sure what "these generally track" points to.

Please rewrite so we have three sentences:

- One sentence about tracking publicly release database versions
- One sentence about how the database versions are (not) released
- One sentence about how database versions can be dropped from the registry